kobo/apps/project_ownership/tests/api/v2/test_api.py

```diff
@@ -500,17 +501,79 @@ def test_data_accessible_to_new_user(self):
         for attachment in response.data['results'][0]['_attachments']:
             assert attachment['filename'].startswith('anotheruser/')
 
+        # Get the mongo_uuid for the transferred asset (XForm)
+        xform = XForm.objects.get(kpi_asset_uid=self.asset.uid)
```

Comment:
You can use the `XForm` object from the deployment, i.e.: `self.asset.deployment.xform`